.github/workflows/dp-workflow.yml

```diff
@@ -55,12 +59,28 @@ jobs:
             helm:
               - '.github/workflows/dp-workflow.yml'
               - 'dp/cloud/helm/**'
-            integration_tests_no_orc8r:
+            integration_tests:
+              - '.github/workflows/**'
+              - 'orc8r/**'
+              - 'lte/**'
               - 'dp/**'
+              - 'nms/**'
+              - 'src/**'
+              - 'scripts/**'
+              - 'protos/**'
             db:
               - '.github/workflows/dp-workflow.yml'
               - 'dp/cloud/python/magma/db_service/**'
-
+      - name: Save should_not_skip output
```

Comment:
Why is this named "Save should_not_skip"? Why is the information from the uploaded file needed?

Reply:
I am not sure of that, other workflows which are using "Unit test reporting" workflow have the same step. I just copied it. It's required to artifact pr.zip exists in Workflow, because on that condition other steps are run from it.

Reply:
Thank you for clarifying. The naming still confuses me - but I think this is easier to change in another PR if this is consistent for now.